build.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const async = require('async');
+
+const getText = (type, cb) => {
+  const files = [
+    'change_password',
+    'create',
+    'delete',
+    'get_user',
+    'login',
+    'verify'
+  ];
+
+  const results = {};
+  async.map(
+    files,
+    (filename, next) => {
+      fs.readFile(`./src/scripts/${type}/${filename}.js`, 'utf8', (err, text) => {
+        results[filename] = text;
+        next(err);
+      });
+    },
+    () => {
+      cb(results);
+    });
+};
+
+const buildAll = () => {
+  fs.mkdir('./dist', () =>
+    fs.readdir('./src/scripts', (err, dirs) => {
+      async.map(
+        dirs,
+        (type, next) => {
+          getText(type, (data) => {
+            const template =
+              `module.exports = {
+                change_password: (${data.change_password}).toString(),
```

Comment:
why apply `toString` after? could this just be:

```
change_password: data.change_password.toString()
```

Reply:
might make sense to use the `files` array above, and build the object dynamically